### DISCUSSION TITLE
interface to receive logs, but not be the management interface?

### DISCUSSION THREAD

**Discussion by j5jb3x4eys** [Security-Onion-Solutions/securityonion#8177] [Unsupported Versions]
2.3.130 - up to date

is it possible to have an interface on SO that can receive logs (winlogbeat, etc.) but not the management interface where analysts access kibana, etc.?

thank you

**Comment by cm-ops** [comment]
Setting the `url_base` to hostname to connect to SOC should work with the additional interface.

**Reply by j5jb3x4eys** [reply]
thank you
